src/Cli/dotnet/Telemetry/TopLevelCommandNameAndOptionToLog.cs

```diff
@@ -24,23 +24,19 @@ internal class TopLevelCommandNameAndOptionToLog : IParseResultLogRule
 
         public List<ApplicationInsightsEntryFormat> AllowList(ParseResult parseResult)
         {
-            var topLevelCommandName = parseResult[DotnetName]?.AppliedOptions?.FirstOrDefault()?.Name;
+            var topLevelCommandName = parseResult.RootCommandResult.Children.FirstOrDefault()?.Symbol.Name;
```

Comment:
This could be an option or argument but it looks like we're expecting a command.